README.md

```
@@ -1,26 +1,30 @@
     ystdlib::<lib_1> ystdlib::<lib_2> ... ystdlib::<lib_N>
     # other libs...
 )
 ```
-Ensure that `ystdlib-cpp` is either within a subdirectory of the folder containing `CMakeLists.txt`
-or at the same level.
 
 # Contributing
 Follow the steps below to develop and contribute to the project.
 
 ## Requirements
+* CMake 3.23 or higher
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)